assets/blocks/DeadBush.block

```diff
@@ -19,9 +19,9 @@
  */
 {
     "author" : "eleazzaar",
-    "basedOn" : "core:plant",
+    "basedOn" : "CoreAssets:plant",
     "entity": {
         "prefab": "PlantPack:DeadBush"
     },
-    "tile": "Core:DeadBush"
+    "tile": "CoreAssets:DeadBush"
 }
```

Comment:
Though not directly related to this PR, adding a newline at end of the file would be good 👍